packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -625,7 +722,7 @@ export const DiscoverGrid = ({
         <EuiText size="xs" color="subdued">
           <EuiLoadingSpinner />
           <EuiSpacer size="s" />
-          <FormattedMessage id="discover.loadingResults" defaultMessage="Loading results" />
+          <FormattedMessage id="unifiedDataTable.loadingResults" defaultMessage="Loading results" />
```

Comment:
Tiny nit, there's a `data-test-subj` called `discoverDataGridLoading` on L721 which could be relabeled, an it seems it's just used here:
https://github.com/elastic/kibana/blob/edef2e2d40a1589f8f1cf1ce45b9994811ef0130/x-pack/test/security_solution_cypress/cypress/screens/discover.ts#L48

just saying since you're doing such a great cleanup and renaming